src/Illuminate/Auth/Access/Gate.php

```diff
@@ -326,7 +326,7 @@ public function after(callable $callback)
      * Determine if all of the given abilities should be granted for the current user.
      *
      * @param  iterable|\UnitEnum|string  $ability
-     * @param  array|mixed  $arguments
+     * @param  mixed  $arguments
```

Comment:
This could be further narrowed down:
```suggestion
     * @param  array{class-string|object, ...} $arguments
```

Reply:
See #56426